pytest/tests/loadtest/locust/common/inscription.py

```diff
@@ -0,0 +1,109 @@
+import logging
+from concurrent import futures
+import random
+import string
+import sys
+import pathlib
+import typing
+from locust import events, runners
+import common.base as base
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
+
+import key
+from common.base import Account, Deploy, NearNodeProxy, NearUser, FunctionCall, INIT_DONE
+from configured_logger import new_logger
+
+logger = new_logger(level=logging.INFO)
+
+
+class MintInscription(FunctionCall):
+
+    def __init__(self, contract_account_id: str, sender: Account, tick, amt):
+        # Attach exactly 1 yoctoNEAR according to NEP-141 to avoid calls from restricted access keys
```

Comment:
I don't see where we attach 1 yoctoNEAR here...

Reply:
From author:
> oh, this is only a comment i got while copying some code/mehod from another locust file…forgot to delete it …you can delete it

I'll remove it